model/src/service/tasks.py

```diff
@@ -21,23 +21,16 @@
 import uuid
 import random
 import requests
-import subprocess
-from rq import Connection, Queue
 from src.service.util import gen_dict_extract
 from src.service import path
-from src.service.backend import initialize_redis
 
-
-DEFAULT_STATUS_ADDRESS = "http://{0}.edge-namespace:80/status" 
+DEFAULT_STATUS_ADDRESS = "http://{0}.edge-namespace:80/status"
 DEFAULT_ROOT_ADDRESS = "http://{0}.edge-namespace:80/"
```

Comment:
We decided to go for dynamic namespace. Here it is hardcoded.

Reply:
yes, this will also be fixed in issue #28.